Deployment/helm/orchestrator/templates/deployment.yaml

```diff
@@ -13,8 +13,20 @@ spec:
         app: orchestrator
     spec:
       containers:
-      - image: nginx:1.15.9
+      - image: {{ .Values.image.repository }}/orchestrator:{{ .Values.image.tag}}
         name: orchestrator
+        env:
+        - name: "GENERATE_MESSAGES_EVERY"
+          value: {{ .Values.GENERATE_MESSAGES_EVERY }}
+        - name: "NUM_MESSAGES_EACH_GENERATION"
+          value: {{ .Values.NUM_MESSAGES_EACH_GENERATION }}
+        - name: "CACHE_GRPC_ENDPOINT"
+          value: {{ .Values.CACHE_GRPC_ENDPOINT }}
```

Comment:
Can you put these value templates in quotes? Without being in quotes, `kubectl apply` will fail to parse the deployment.yaml Helm produced.

Reply:
Done if looks good please help to merge